.github/workflows/e2e-test.yaml

```diff
@@ -19,14 +22,41 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
 
-      - name: Setup Kind cluster
-        id: setup-kind
+      - name: Setup Kind cluster1
+        id: setup-kind1
         uses: ./.github/actions/kind-create
+        with:
+          kind_cluster_name: kind1
+          kind_k8s_version: v1.24.13
+          kubeconfig_dir_path: tests/e2e/kubeconfigs
+
+      # TODO: ZookeeperCluster create time out always on the second kind cluster
```

Comment:
Question: Is it an actual timeout issue or can Zookeeper not start for some reason and we retry until we reach a timeout?

Reply:
Here you can see the problem: https://github.com/banzaicloud/koperator/runs/15416787902
It would be nice to get more information why is this fail (like pod status)

Reply:
Because in local environment all of the tests passed when multiple k8s cluster is used in parallel way I suspect that the problem is with timeouts or something else on github

Reply:
Please test the PR with multicluster input on your local. 
This PR is not about to run tests on github on multiple kind cluster.